common/lib/xmodule/xmodule/graders.py

```diff
@@ -13,18 +19,25 @@
 Score = namedtuple("Score", "earned possible graded section module_id")
 
 
+def float_sum(iterable):
+    """
+    Sum the elements of the iterable, and return the result as a float.
+    """
+    return float(sum(iterable))
```

Comment:
Just to cover all the bases here - do we need to cast each individual element of `iterable` as a float before summing?